### PR TITLE
Fuchsia: convert framebuffer pixel format after drawing if needed

### PR DESCRIPTION
**Pull request by mikejurka**

### CODE REVIEW

File: content_handler/framebuffer_skia.cc [hunk 1]
```diff
@@ -88,4 +88,25 @@ void FramebufferSkia::Bind(mojo::InterfaceHandle<mojo::Framebuffer> framebuffer,
...
+    if (surface_->peekPixels(&bufferPixmap)) {
+      uint8_t* buffer = reinterpret_cast<uint8_t*>(bufferPixmap.writable_addr());
+      size_t buffer_size = bufferPixmap.getSafeSize();
+      uint8_t src_r;
+      for (size_t i = 0; i < buffer_size; i++) {
+        src_r = buffer[0];
+        buffer[0] /* A */ = buffer[3];
+        buffer[3] /* B */ = buffer[2];
+        buffer[2] /* G */ = buffer[1];
+        buffer[1] /* R */ = src_r;
```
Comment (abarth) [inline content_handler/framebuffer_skia.cc]: I wonder if there's some fancy assembly instruction to rotate these bytes.  This is fine too.

**Comment by abarth** [comment]
LGTM

**Comment by mikejurka** [comment]
I haven't tested it on an Acer (only verified it can change colors on QEMU), but I'll merge. Let me know if I got the mapping wrong 😜

File: content_handler/framebuffer_skia.cc (hunk 1 above)
Comment (mikejurka) [inline content_handler/framebuffer_skia.cc]: yes, I wondered too. I found https://github.com/gwiazdorrr/CxxSwizzle which supposedly uses SIMD but 1) didn't want to add a dependency just for this and 2) wasn't sure if it did exactly what we need.

In the end, decided to do the simplest thing just to get something in. Will be curious how it affects performance.

Comment (mikejurka) [inline content_handler/framebuffer_skia.cc]: I can give it a try with CxxSwizzle or something else later too.

**Pull request merged**